bee-network/bee-autopeering/src/event.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Events published to the user.
+
+use crate::{
+    peer::{Peer, PeerId},
+    peering::neighbor::Distance,
+};
+
+use tokio::sync::mpsc;
+
+use std::fmt;
+
+/// Autopeering related events.
+#[derive(Debug)]
+pub enum Event {
+    /// A new peer has been discovered.
+    PeerDiscovered {
+        /// The discovered peer.
```

Comment:
```suggestion
        /// The discovered peer identity.
```

Reply:
Doesn't discovering an identity imply discovering a peer, so why being so specific here?

Reply:
It's just inconsistent with your other comments

Reply:
okay, will make this consistent then :+1:

Reply:
Done